src/util/db/dbconnectionpool.cpp

```diff
@@ -71,16 +71,20 @@ DbConnectionPool::DbConnectionPool(
       m_connectionCounter(0) {
 }
 
-DbConnectionPool::ThreadLocalScope::ThreadLocalScope(
+DbConnectionPool::ThreadLocalScoped::ThreadLocalScoped(
```

Comment:
DbConnectionPool::ThreadLocalScoped is IMHO still not self explaining. Isn't it a kind of Scoped pointer for a DbConnection? Normally we would call it DBConnectionPtr without the smart pointer type Hint. It is OK for me to add all the additional features to the name, if if we can efford the extra characters so it is  DBConnectionThreadLocalScopedPtr.